cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/remote/RemoteLineageWriterTest.java

```diff
@@ -14,14 +14,13 @@
  * the License.
  */
 
-package co.cask.cdap.internal.app.store;
+package co.cask.cdap.internal.app.store.remote;
```

Comment:
Some of these classes are package private and thus moving these classes seem to break things. I am going to leave it as it is, unless there is a strong desire to change it in this PR.